src/pytorch_lightning/core/module.py

```diff
@@ -50,7 +50,8 @@
 from pytorch_lightning.trainer.connectors.logger_connector.fx_validator import _FxValidator
 from pytorch_lightning.utilities import GradClipAlgorithmType
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.imports import _TORCH_GREATER_EQUAL_1_13
```

Comment:
You need to keep this import. Not Fabric's

Because HPU does modify it: https://github.com/Lightning-AI/lightning/blob/a56c12c26f8f77c22984b744cf1d830a3c813cfb/src/pytorch_lightning/utilities/imports.py#L48-L49

Reply:
thanks, that actually makes a lot of sense. I don't know how I could overlook this one. 🤦